MzTab_MTD

```diff
@@ -168,10 +227,322 @@ CREATE TABLE "MzTab"."PSM Section"
+    
+    
+-- Table: XML.TraML
+
+-- DROP TABLE "XML"."TraML";
+
+CREATE TABLE "XML"."TraML"
+(
+    "id " text COLLATE pg_catalog."default",
+    "version " text COLLATE pg_catalog."default" NOT NULL,
+    "cvList" text[] COLLATE pg_catalog."default",
+    "SourceFileList" text[] COLLATE pg_catalog."default",
+    "ContactList" text[] COLLATE pg_catalog."default",
+    "PublicationList" text[] COLLATE pg_catalog."default",
+    "InstrumentList" text[] COLLATE pg_catalog."default",
+    "SoftwareList" text[] COLLATE pg_catalog."default",
+    "ProteinList" text[] COLLATE pg_catalog."default",
+    "CompoundList" text[] COLLATE pg_catalog."default",
+    "TransitionList" text[] COLLATE pg_catalog."default",
+    "TargetList" text[] COLLATE pg_catalog."default"
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE "XML"."TraML"
+    OWNER to postgres;
+    
+  
+ 
+-- Table: XML.TraML
+
+-- DROP TABLE "XML"."TraML";
+
+CREATE TABLE "XML"."TraML"
+(
+    "GeneName" text COLLATE pg_catalog."default",
+    "ProteinName" text COLLATE pg_catalog."default",
+    "PeptideSequence" text COLLATE pg_catalog."default",
+    "FullPeptideName" text COLLATE pg_catalog."default",
+    transition_group_id text COLLATE pg_catalog."default",
+    transition_name text COLLATE pg_catalog."default",
+    "Annotation" text COLLATE pg_catalog."default",
+    "PrecursorMz" double precision,
+    "MS1 Res" text COLLATE pg_catalog."default",
+    "ProductMz" double precision,
+    "MS2 Res" text COLLATE pg_catalog."default",
+    "Dwell" integer,
+    "Fragmentor" integer,
+    "CollisionEnergy" double precision,
+    "CellAcceleratorVoltage" integer,
+    "LibraryIntensity" double precision,
+    decoy integer,
+    "LabelType" text COLLATE pg_catalog."default",
+    "PrecursorCharge" integer,
+    "FragmentCharge" integer,
+    "FragmentType" text COLLATE pg_catalog."default",
+    "FragmentSeriesNumber" integer,
+    quantifying_transition boolean,
+    identifying_transition boolean,
+    detecting_transition boolean,
+    "Tr_recalibrated" double precision,
+    "RetentionTime" double precision,
+    "PeptideGroupLabel" text COLLATE pg_catalog."default",
+    "UniprotId" text COLLATE pg_catalog."default",
+    "CompoundName" text COLLATE pg_catalog."default",
+    "SMILES" text COLLATE pg_catalog."default",
+    "SumFormula" text COLLATE pg_catalog."default",
+    "mzTab-ID" text COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT "TraML_pkey" PRIMARY KEY ("mzTab-ID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE "XML"."TraML"
+    OWNER to postgres;
+
+-- Table: XML.FeatureXML
+
+-- DROP TABLE "XML"."FeatureXML";
+
+CREATE TABLE "XML"."FeatureXML"
+(
+    "Investigation_ID" text COLLATE pg_catalog."default" NOT NULL,
+    "Study_ID" text COLLATE pg_catalog."default" NOT NULL,
+    "Assay_ID" text COLLATE pg_catalog."default" NOT NULL,
+    "mzTab-ID" text COLLATE pg_catalog."default" NOT NULL,
+    "Sample_Name" text COLLATE pg_catalog."default" NOT NULL,
```

Comment:
It looks like a column for the `FeatureMap` with type XML is missing.

Reply:
i added FeatureMap